### PR TITLE
RANCHER-1427. Correct ingress path for the edge-dcb module.

### PR DESCRIPTION
**Pull request by epam-avramenko**

### CODE REVIEW

File: charts/edge-dcb/values.yaml [hunk 1]
```diff
@@ -49,7 +49,7 @@ ingress:
   hosts:
     - host: chart-example.local
       paths:
-        - path: /*
+        - path: /dcbService*
```
Comment (eldiiar-duishenaliev) [inline charts/edge-dcb/values.yaml:52]: updating chart version isn't required in this case? How indexing of chart will work with this change?

Comment (epam-avramenko) [reply charts/edge-dcb/values.yaml:52]: Agree. My bad. Have changed.

Comment (aatoyan) [reply charts/edge-dcb/values.yaml:52]: If I'm not mistaken common charts should also be updated

Comment (epam-avramenko) [reply charts/edge-dcb/values.yaml:52]: There is no need to change the folio-common chart version as no changes have been made. The approach, "the more we hand over, the better it is" is possible but not required.

**Review by aatoyan — APPROVED** [review]

**Review by OHaimanov — APPROVED** [review]

**Pull request merged**